config/swagger.yaml

```diff
@@ -613,7 +612,7 @@ paths:
         '200':
           description: 'Successful operation'
           schema:
-            $ref: '#/definitions/ContractCallObject'
+            $ref: '#/definitions/TxInfoObject'
```

Comment:
Worth a line in release notes.

Reply:
Yes, release notes are TODO. I want to do it in a separate PR, but sure, needs doing.

Reply:
I see release notes added. Thanks.